manifests/ha/install.yaml

```diff
@@ -10792,7 +10792,7 @@ spec:
               key: reposerver.log.format
               name: argocd-cmd-params-cm
               optional: true
-        - name: ARGOCD_REPO_SERVER_LOGLEVEL
+        - name: ARGOCD_SERVER_LOG_LEVEL
```

Comment:
You might have to run `make manifests` or `make manifests-local` to clear this up. :-)